app/src/main/res/layout/item_status.xml

```diff
@@ -108,23 +108,42 @@
             android:textColor="?android:textColorPrimary"
             android:textSize="?attr/status_text_medium" />
 
-        <ToggleButton
-            android:id="@+id/status_content_warning_button"
+        <RelativeLayout
+            android:id="@+id/status_content_warning_button_area"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginBottom="6dp"
-            android:background="?attr/content_warning_button"
-            android:minHeight="0dp"
-            android:minWidth="0dp"
-            android:paddingBottom="3dp"
-            android:paddingLeft="6dp"
-            android:paddingRight="6dp"
-            android:paddingTop="3dp"
-            android:textAllCaps="true"
-            android:textOff="@string/status_content_warning_show_more"
-            android:textOn="@string/status_content_warning_show_less"
-            android:textSize="?attr/status_text_small" />
-
+            android:layout_below="@+id/status_content_warning_description"
+            android:layout_marginBottom="4dp"
+            android:layout_toEndOf="@+id/status_avatar"
+            android:focusable="true"
+            android:paddingTop="6dp"
+            android:paddingBottom="6dp"
+            app:paddingHorizontal="4dp">
+            <RelativeLayout
+                android:id="@+id/status_content_warning_button_center_container"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                >
+
+                <ToggleButton
+                    android:id="@+id/status_content_warning_button"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_centerHorizontal="true"
+                    android:background="?attr/content_warning_button"
+                    android:minHeight="0dp"
+                    android:minWidth="0dp"
+                    android:paddingBottom="6dp"
+                    android:paddingLeft="10dp"
+                    android:paddingRight="10dp"
+                    android:paddingTop="6dp"
+                    android:textAllCaps="true"
+                    android:textOff="@string/status_content_warning_show_more"
+                    android:textOn="@string/status_content_warning_show_less"
+                    android:textSize="?attr/status_text_large" />
```

Comment:
`?attr/status_text_medium` should be enough? Maybe add some more padding in exchange?